src/components/CommentVoter.jsx

```diff
@@ -0,0 +1,44 @@
+import { useState } from "react";
+import { patchCommentVotes } from "../utils";
+
+const CommentVoter = ({ votes, comment_id }) => {
+  const [voteIncrement, setVotes] = useState(0);
+  const [err, setErr] = useState(null);
+
+  const handleVoteIncrease = () => {
```

Comment:
You could condense both of these functions into handleVoteChange and pass in the direction that it should go in